playwright/async_api.py

```diff
@@ -2928,6 +2928,22 @@ async def path(self) -> typing.Union[str, NoneType]:
         """
         return mapping.from_maybe_impl(await self._impl_obj.path())
 
+    async def createReadStream(
+        self, fp: _io.BytesIO = None
+    ) -> typing.Union[_io.BytesIO, NoneType]:
+        """Download.createReadStream
+
+        Returns readable stream for current download or `null` if download failed.
+
+        Parameters
```

Comment:
No parameters because we could not find fp in api.json. We should still list those w/o docs...